Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs

```diff
@@ -12,14 +12,17 @@ FormattingContext context
         var docs = new List<Doc> { Token.Print(openParenToken, context) };
 
         if (
-            arguments.Count == 1
-            && arguments[0].Expression
-                is ParenthesizedLambdaExpressionSyntax
-                    {
-                        ParameterList.Parameters.Count: 0,
-                        Block: { }
-                    }
-                    or SimpleLambdaExpressionSyntax { Block: { } }
+            arguments
+            is [
+                {
+                    Expression: ParenthesizedLambdaExpressionSyntax
```

Comment:
oooh. Fancy list pattern match syntax :beach_umbrella:

Reply:
that was all rider...